shared_model/utils/string_builder.hpp

```diff
@@ -0,0 +1,119 @@
+/**
```

Comment:
I think it's better place this class in common folder instead of shared_model

Reply:
Well, after discussion with @muratovv we decided to leave it inside shared model.